skl2onnx/_supported_operators.py

```diff
@@ -216,8 +234,8 @@ def build_sklearn_operator_name_map():
                 GenericUnivariateSelect, RFE, RFECV, SelectFdr, SelectFpr,
                 SelectFromModel, SelectFwe, SelectKBest, SelectPercentile,
                 VarianceThreshold, GaussianMixture, GaussianProcessRegressor,
-                BayesianGaussianMixture, OneClassSVM,
-                PLSRegression
+                BayesianGaussianMixture, OneClassSVM, PLSRegression,
+                HistGradientBoostingClassifier, HistGradientBoostingRegressor
```

Comment:
Same comment as above.

Reply:
We should short them all bug that's not the only place this is done like that.